doc-content/enterprise-only/cases/case-management-creating-roles-proc.adoc

```diff
@@ -1,6 +1,8 @@
 [id='case-management-creating-roles-proc-{context}']
 = Creating case roles
 
+*<MH>Dev, in viewing the IT_Orders case project in 7.6, I see that the case definitions are actually referred to as business processes when viewing the sample assets. Should we consider a change in the naming of the assets moving forward to avoid confusion. We are using the term "Case definition" as if it will be listed somewhere.</MH>*
```

Comment:
Case definitions are like normal business processes, so that's why they are listed as such in the asset list.  The difference is that if you create a case definition, it creates a business process where the dynamic property is already set to true.

Reply:
Noted. Thank you for the explanation. I'll ask you the same thing for every future release. Kidding.